tests/cache/tests.py

```diff
@@ -146,6 +146,11 @@ def test_decr(self):
         with self.assertRaises(ValueError):
             cache.decr('does_not_exist')
 
+    def test_touch(self):
+        "Non existent keys can't be successfully touched"
+        touched = cache.touch('some_key')
+        self.assertFalse(touched)
```

Comment:
Use `assertIs(touched, False)` since `assertFalse` passes even if `bool(touched)` is `False`.